### ISSUE TITLE
Settings PropertyEditorUI for mapping fields.

### ISSUE THREAD

**Issue by enkelmedia** [umbraco/Umbraco.Forms.Issues#1337]
I tend to often needs a way to map known properties against fields in the form.

Some examples:

**Integrate with CRM X**
In this scenario we know that we want to collect Firstname, Lastname, Company and Email, on the workflow I would like to offer the editor to choose the mapping in a nice way.

**Integrate with Campaign tool Y**
In this scenario we know that we want to collect Name and Email, on the workflow I would like to offer the editor to choose the mapping in a nice way.

It would be really nice if there was a Property Editor UI that we could feed with these "known" values and that would show them together with a form field dropdown so the editor can "map them".

Basically a table like so:

| Property | Field |
|-----|-----|
| Firstname | --Choose Field-- |
| Lastname | --Choose Field-- |
| Company | --Choose Field-- |
| Email | --Choose Field-- |

I did look at the docs https://docs.umbraco.com/umbraco-forms/developer/extending/adding-a-type/setting-types but as I understand it there is no such thing today?
